Export.h.in

```diff
@@ -0,0 +1,138 @@
+/** @file
+    @brief Automatically-generated export header - do not edit!
+
+    @date 2016
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+// Copyright 2016 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef @EXPORT_BASENAME@_EXPORT_H
+#define @EXPORT_BASENAME@_EXPORT_H
+
+#ifdef @EXPORT_BASENAME@_STATIC_DEFINE
+#  define @EXPORT_BASENAME@_EXPORT
+#  define @EXPORT_BASENAME@_NO_EXPORT
+#endif
+
+/* Per-compiler advance preventative definition */
+#if defined(__BORLANDC__) || defined(__CODEGEARC__) || defined(__HP_aCC) ||    \
+    defined(__PGI) || defined(__WATCOMC__)
+/* Compilers that don't support deprecated, according to CMake. */
+#  ifndef @EXPORT_BASENAME@_DEPRECATED
+#    define @EXPORT_BASENAME@_DEPRECATED
+#  endif
+#endif
+
+/* Check for attribute support */
+#if defined(__INTEL_COMPILER)
+/* Checking before GNUC because Intel implements GNU extensions,
+ * so it chooses to define __GNUC__ as well. */
+#  if __INTEL_COMPILER >= 1200
+/* Intel compiler 12.0 or newer can handle these attributes per CMake */
+#    define @EXPORT_BASENAME@_EXPORT_HEADER_SUPPORTS_ATTRIBUTES
+#  endif
+
+#elif defined(__GNUC__)
+#  if (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 2))
+/* GCC 4.2+ */
+#    define @EXPORT_BASENAME@_EXPORT_HEADER_SUPPORTS_ATTRIBUTES
+#  endif
+#endif
+
+/* Per-platform defines */
+#if defined(_MSC_VER)
+/* MSVC on Windows */
+
+#ifndef @EXPORT_BASENAME@_EXPORT
+#  ifdef @LIBNAME_FULL@_EXPORTS
```

Comment:
We also need to `set(LIBNAME_FULL osvrRenderManager)` before the configure file command, so that the file knows if we're building or using the library.